akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -586,7 +587,7 @@ import scala.util.control.NonFatal
 
   private[stream] def fail(connection: Connection, ex: Throwable): Unit = {
     val currentState = connection.portState
-    if (Debug) println(s"$Name   fail($connection, $ex) [$currentState]")
+    if(alwaysLogErrors) log.error(ex, "Error in stage [{}]: {}", activeStage.originalStage.getOrElse(activeStage), ex.getMessage)
```

Comment:
the Debug one is somewhat separate I think and we may want to keep it -- it's formatting helps if you actually use the Debug mode. So this should be a separate if / log statement as the purpose is different